draft-ietf-scitt-architecture.md

```diff
@@ -326,6 +326,15 @@ The SCITT architecture consists of a very loose federation of Transparency Servi
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Signed Inclusion Proofs.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
+{{fig-concept-relationship}} illustrates two Transparency Services and two Receipts as a single Signed Statement MAY be registered on more than one Transparency Service.
+Each Transparency Service produces a Receipt.
+A Transparent Statement MAY contain multiple Receipts, for each Transparency Service it was successfully Registered.
+
+The arrows indicate the flow of information.
+For example, Credentials are used to Sign Statements and Verify Signed Statements within the Transparency Service.
+The Transparency Service produces a Receipt, which MAY be combined with a Signed Statement to create a Transparent Statement.
+To Verify a Transparent Statement, credentials are required to authenticate the protected content of the Transparent Statement.
```

Comment:
```suggestion
To verify a Transparent Statement, credentials are required to authenticate the protected content of the Transparent Statement.
```

Reply:
Which implies that you actually checking the signature of the Signed Statement?

Would "verifying" a Transparent Statement not mean to check Signed Statement signature and Receipt signature?